pymc_marketing/mmm/delayed_saturated_mmm.py

```diff
@@ -433,17 +433,17 @@ def build_model(
                     "baseline_intercept"
                 )
 
-                intercept_latent_process = create_time_varying_gp_multiplier(
-                    name="intercept",
-                    dims="date",
-                    time_index=time_index,
-                    time_index_mid=self._time_index_mid,
-                    time_resolution=self._time_resolution,
-                    hsgp_kwargs=self.model_config["intercept_tvp_config"],
+                intercept_latent_process = (
+                    self.model_config["intercept_tvp_config"]
+                    .register_data(time_index)
+                    .create_variable(
+                        name="intercept_temporal_latent_multiplier",
+                        dims="date",
+                    )
                 )
                 intercept = pm.Deterministic(
                     name="intercept",
-                    var=baseline_intercept * intercept_latent_process,
+                    var=pt.exp(baseline_intercept + intercept_latent_process),
```

Comment:
I'll add the same comment than before here:

Can we deep-dive a bit more about this implementation?

In the original implementation, The mean of 𝛽𝑡 would be the
**_beta (since the mean of multiplier is 1). Then variance of 𝛽𝑡 will depend on the variance of the multiplier.

Because of it, the initial implementation is more straightforward and interpretable (Personal opinion). Mostly around the response curves.

What would be the benefit of this parameterization, if it is not more interpretable?
Being exponential, it forces the final parameter to be positive. This depending on the parameter we want to be time dependent may or may not be needed.

I think that if the result of **_base + latent is a normal distribution, there should be no problem, the result is a LogNormal distribution. But if it is Gamma or HalfNormal, the result would be an even more skewed distribution -This might be feasible because the config controls the base distributions- I'm not sure if it's a problem or not, but have we thought about how this transformation would apply to different arbitrary distributions?

A couple of questions that may be very obvious and I am a little slow to see the answers! 😅